frontend/src/forms/subforms/BuildingForm.tsx

```diff
@@ -189,7 +189,7 @@ const BuildingForm = <T extends any>(props: BuildingProps & FormikProps<T>) => {
               RAEG or SPP
             </Form.Label>
             <Input
-              disabled={true}
+              disabled={props.disabled}
```

Comment:
last I heard project number was not supposed to be editable on this form, has that changed?

Reply:
Yeah, I have a user story for it. I believe it was after a convo Milkaela and Jeremy had. In order to get the existing RAEG/SPPs in the system

Reply:
Only if the user has `property-admin` claim though I believe.